recipes/tomviz/conda_build_config.yaml

```diff
@@ -0,0 +1,16 @@
+channel_sources:
+  - nsls2forge,defaults,conda-forge
+channel_targets:
+  - nsls2forge main
+cdt_name:
+  - cos6
+c_compiler_version:            # [unix]
+  - 7                          # [linux]
+cxx_compiler_version:          # [unix]
+  - 7                          # [linux]
+numpy:
+  - 1.16
+python:
+  - 3.7
+python_impl:
+  - cpython
```

Comment:
This file is not needed for conda-forge (will use all default values). Please rerender after removing this file.

Reply:
I have removed the file, it doesn't look like anyone renders the recipe on stage?

Reply:
I think you are right, no rerender is needed in this step (at least I don't see I rerendered anything in https://github.com/conda-forge/staged-recipes/pull/10819).